com.jetbrains.WebStorm.yaml

```diff
@@ -24,21 +25,15 @@ modules:
         url: https://nodejs.org/dist/v10.15.0/node-v10.15.0.tar.xz
         sha256: 797ab34c74b83b21b7d6ea261b5ca235d34c61a7da5aebb32459a963097ede3e
 
-  - name: WebStorm
+  - name: webstorm
     buildsystem: simple
     build-commands:
       - unzip webstorm_logos.zip
       - install -D apply_extra /app/bin/apply_extra
-      - install -Dm755 webstorm-desktop.sh "${FLATPAK_DEST}/bin/webstorm-desktop"
-      - install -Dm644 logo.svg "${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg"
-      - icon_in=logo.svg;
```

Comment:
@FakeShemp, I removed the whole `rsvg-convert` Jiu Jitsu. I don't think it's needed, but I may be wrong. I see the correct icons when I provide the `SVG`. Let me know in any case :sunglasses:

Reply:
I'm not sure all distros handle svg-only icons. I would prefer to only use them, but others seem to object. Maybe someone knowledgeable can give some insight.

Reply:
What I meant was that, somehow, Flathub builds generate the appropriate `PNG` files/icons (in the correct locations) when you provide an `SVG`; take a look at the build's log. Now, it doesn't generate all the resolutions :innocent: